docker-entrypoint.sh

```diff
@@ -128,7 +128,7 @@ su - $RUN_AS -c "s3fs $DEBUG_OPTS ${S3FS_ARGS} \
 
 # s3fs can claim to have a mount even though it didn't succeed. Doing an
 # operation actually forces it to detect that and remove the mount.
-su - $RUN_AS -c "ls ${AWS_S3_MOUNT}"
+su - $RUN_AS -c "touch ${AWS_S3_MOUNT}/.s3fs-test.txt && rm ${AWS_S3_MOUNT}/.s3fs-test.txt"
```

Comment:
This will fail on read-only mounts.  Could you run `stat` instead?

Reply:
Fair enough. Would running `stat` on the mount point do the trick?